doc/example-config

```diff
@@ -1018,15 +1007,26 @@
 #
 # prepopulate-query-feeds yes
 
-####  ssl-verify
+####  ssl-verifyhost
+#
+# If no, skip to verify the certificate's name against host.
+#
+# Syntax: [yes/no]
+#
+# Default value: yes
+#
+# ssl-verifyhost no
+
+####  ssl-verifypeer
 #
-# If no, skip SSL certificate validation.
+# If no, skip to verify the peer's SSL certificate which can be used to avoid errors
```

Comment:
"Skip to verify" isn't the same as "skip verification". If you imagine a numbered list of actions and, say, the 6th item said "verify certificate" and 3rd item said "skip to verify", this would mean that after the 3rd step you would execute the 6th, i.e. you would skip over items 4 and 5. If 3rd item said "skip verification" instead, this would mean that after 3rd step you would execute 4th and 5th, but you won't execute 6th (because 3rd told you to skip it).

> If no, skip verification of the peer's SSL certificate …